app/views/arrange-a-session/add-notes.html

```diff
@@ -0,0 +1,29 @@
+{% extends "_wizard-form.html" %}
+{% set title = "Would you like to add notes about this appointment?" %}
+{% set buttonText = 'Save and continue' %}
+
+{% block form %}
+  {{ govukRadios({
+       fieldset: {
+         legend: {
+           text: title,
+           classes: "govuk-label--xl",
+           isPageHeading: true
+         }
+       },
+       hint: {
+         html: 'Notes can also be added later'
+       },
+       items: [
+         {
+           text: 'Yes',
+           value: 'Yes'
+         },
```

Comment:
Just as a note – if you leave out the value it will default to the text, so you could simplify to:

```suggestion
         {
           text: 'Yes'
         },
```

But I wonder if the labels should be:

- Yes, add a note
- No

I like to err on one of the options being clearer than yes/no so you're not solely relying on how the user has read and interpreted the question.

Reply:
Agree with the above

For both arrange an appointment and confirm attendence:
- Removed the values from the `textarea`
- Updated the radio button labels to make the answer clearer to the user
- Due to this, I've also removed the hint text as it no longer seemed necessary

**Before**
![115745509-67ab6700-a38b-11eb-8d9d-f9f74e1d3d5d](https://user-images.githubusercontent.com/6122118/115841537-cc5ed400-a414-11eb-8bca-bebc2757c3fe.png)

**After**
![Screenshot 2021-04-23 at 09 15 07](https://user-images.githubusercontent.com/6122118/115841558-d1bc1e80-a414-11eb-9d75-8e186dcd9f71.png)

Reply:
Not sure about "I'll do this later", as they won't be doing it later if they have no notes to add.

```
- Yes, add appointment notes
- No
```
?

Reply:
Agree with your point.

How about:
`- Yes
- No (Notes can be added later)`
?